rfc-009-transparent-code-bases.md

```diff
@@ -0,0 +1,20 @@
+# Transparent Codebases
+
+## Summary
+
+Teams are expected to publish details on accessing their source code.
+
+## Problem
+
+It is currently difficult for teams to see what other teams are working on, and gain insights, provide feedback and generally for us as a company to learn from each other.
+
+Even in the case where projects are hosted on madetech github, it's hard to filter through the large number of repositories.
```

Comment:
How will having more transparency (and potentially even more forked repositories) help with this? It is already hard to filter through the large number of them, but I'm not sure how having links to what I assume would be very similar to the readmes in a centralised location would make this easier to navigate?

Reply:
Google Spreadsheet:

| Customer | Link to readme |
|---|---|
| Akzo eCom | click here |
| Akzo eFulfilment Hub | click here |

Reply:
I guess to answer the question, I'm less concerned about our github being full of repos, more being able to get signal from the noise.